db/migrate/20210107005114_drop_site_texts.rb

```diff
@@ -0,0 +1,5 @@
+class DropSiteTexts < ActiveRecord::Migration[5.2]
+  def change
+    drop_table :site_texts
```

Comment:
This is an irreversible migration and I think it would be better if it weren't (because I just went to roll it back to work on something else and couldn't).

```suggestion
    drop_table :site_texts do |t|
      t.string 'name'
      t.text 'text'
      t.string 'slug'
      t.timestamps
    end
```